src/test/java/com/epam/tc/hw3/page/DiffElementsPage/DiffElementsPage.java

```diff
@@ -1,7 +1,6 @@
 package com.epam.tc.hw3.page.DiffElementsPage;
```

Comment:
forgot to mention it in hw3 - you should move all your page objects from tests package to src/main/java/(your_packages_for_PO)